lib/virt_feature_test_base.pm

```diff
@@ -71,6 +73,7 @@ sub run {
     $self->run_test;
     $self->{"stop_run"} = time();
 
+    check_guest_health($_) foreach (keys %virt_autotest::common::guests);
```

Comment:
Will running  check_guest_health *once* here for each feature test be enough to tell where the journal problem happens?
BTW, may I ask how long in general will it take for adding this check?

Reply:
> Will running check_guest_health once here for each feature test be enough to tell where the journal problem happens?

No, it is hard to determine which test owns the error because the guest is rebooted in each test. In addition, only the jouranls of current boot will be uploaded, we missed the log as in the PR description.

Reply:
> BTW, may I ask how long in general will it take for adding this check?

It takes 42 seconds in a case.

```
[2024-02-22T16:39:56.946771Z] [debug] [pid:32331] tests/virt_autotest/unified_guest_installation.pm:103 called virt_autotest::utils::check_guest_health -> lib/virt_autotest/utils.pm:319 called testapi::script_run
...
[2024-02-22T16:40:38.913179Z] [debug] [pid:32331] tests/virt_autotest/unified_guest_installation.pm:103 called virt_autotest::utils::check_guest_health -> lib/virt_autotest/utils.pm:331 called testapi::record_info
[2024-02-22T16:40:38.913389Z] [debug] [pid:32331] <<< testapi::record_info(title="Healthy guest!", output="sles-15-sp6-64-fv-uefi-xen looks good so far!", result="ok")
```

Reply:
Thank you for the data.
Is it correct understanding that this PR is not *complete* fix for the problems you described yet?

Reply:
Thank you for the data.
Is it correct understanding that this PR is not *complete* fix for the problems you described yet?

Reply:
It does fix the two problems in the PR description.